meta/runtime.yml

```
@@ -50,45 +50,45 @@ plugin_routing:
   modules:
     aws_az_facts:
       deprecation:
-        removal_date: TBD
+        removal_date: 2022-06-01
         warning_text: see plugin documentation for details
     aws_caller_facts:
       deprecation:
-        removal_date: TBD
+        removal_date: 2022-06-01
```

This is now out of sync with the module
```suggestion
        removal_date: 2021-12-01
```

---

*headdesk* thank you